scripts/py_matter_idl/matter_idl/tests/outputs/cluster_with_commands/bridge/OnOff.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+#include "BridgeGlobalStructs.h"
+#include "third_party/connectedhomeip/examples/dynamic-bridge-app/linux/include/GeneratedClusters.h"
```

Comment:
This path seems very specific ... are there alternatives? Could we have the includes in bridge set up so that just "GeneratedClusters.h" is visible and not rely on the folder structure and that "bridge app" is used?

Reply:
Addressed in [a21b528](https://github.com/project-chip/connectedhomeip/pull/23840/commits/a21b5281a0f74b04907928518683543f618022ce).